<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditAddScore</class>
 <widget class="QDialog" name="AuditAddScore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>加分审核</string>
  </property>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>92</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>260</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>注:不超过3分*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>316</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>加分理由:(注:不超过200字)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>256</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="suffix">
    <string>分</string>
   </property>
   <property name="maximum">
    <double>3.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>96</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>16</y>
     <width>611</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="classEdit">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>142</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>47</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>640</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>346</y>
     <width>401</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>189</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>上学期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>下学期</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>146</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>班级:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>50</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>学号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>196</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>申请学期:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>综测加分:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>640</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
